### PR TITLE
Use regex.test() when we want to check for a Boolean.

### PR DESCRIPTION
**Pull request by XhmikosR**
Not sure if we should backport this

### CODE REVIEW

File: js/src/tooltip.js [hunk 1]
```diff
@@ -747,9 +747,8 @@ class Tooltip {
   _cleanTipClass() {
     const tip = this.getTipElement()
     const tabClass = tip.getAttribute('class').match(BSCLS_PREFIX_REGEX)
-    if (tabClass !== null && tabClass.length) {
-      tabClass
-        .map(token => token.trim())
+    if (tabClass !== null && tabClass.length > 0) {
```
Comment (Johann-S) [inline js/src/tooltip.js]: `tabClass.length` is equal too `tabClass.length > 0` not really needed that change

**Review by Johann-S — APPROVED** [review]
LGTM 👍 

I don't think it's really needed to backport that

File: js/src/tooltip.js (hunk 1 above)
Comment (XhmikosR) [reply js/src/tooltip.js]: It's what we do in poper.js; this code is exactly the same

Comment (Johann-S) [reply js/src/tooltip.js]: So we can change that in `popover.js` too 😄 but it's nitpicking

Comment (XhmikosR) [reply js/src/tooltip.js]: Fine by me, I'd just prefer if we enforced this with a rule, but I cannot find one, only for the opposite `unicorn/explicit-length-check`

**Pull request merged**